docs/openstack/backend_services_deployment.md

```diff
@@ -167,6 +167,8 @@ podified OpenStack control plane services.
       templates:
         openstack:
           storageRequest: 500M
+        openstack-cell1:
+          storageRequest: 500M
```

Comment:
Do we need to support adopting a OSP17 multicell deployment with more than cell1? If so then this needs at least a note above that each cell needs its own DB create here.